fcrepo-webapp/src/main/resources/spring/master.xml

```diff
@@ -3,14 +3,13 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="
     http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd">
-    
+
     <!-- Master context for fcrepo4. -->
 
     <import resource="${fcrepo.spring.repo.configuration:classpath:/spring/repo.xml}"/>
     <import resource="${fcrepo.spring.rest.configuration:classpath:/spring/rest.xml}"/>
-    <import resource="${fcrepo.spring.minter.configuration:classpath:/spring/minter.xml}"/>
```

Comment:
Removing the minter.xml config file results in falling back on the default impl. in the `HierarchicalIdentifierSupplier` interface.  This is fine and preserves the default behavior.

But it also makes it easier to customize the minting.  Instead of being able to customize the number and length of path prefixes with env. variables, you would now have to compile a separate package, create a minter config file to configure it, and then replace/override one of the core config files to include it.

Can we make it easier to override the default minting behavior?  We probably don't want to complicate the default impl, but maybe we can at least make it easier to include custom local config?  E.g., we could have an empty "local" config file to provide an easy place to put custom config.